gui/quantum.lua

```diff
@@ -11,262 +11,218 @@ local quickfort = reqscript('quickfort')
 local quickfort_command = reqscript('internal/quickfort/command')
 local quickfort_orders = reqscript('internal/quickfort/orders')
 
-QuantumUI = defclass(QuantumUI, guidm.MenuOverlay)
+
+-- UI Layout
+QuantumUI = defclass(QuantumUI, gui.ZScreen)
```

Comment:
Try not to mix the functions between two classes. It makes the logic of the file much harder to follow. Put the `ZScreen` subclass at the bottom and the `Window` subclass (and all other business logic) at the top